backend/db/utils.py

```diff
@@ -50,6 +52,24 @@
 SCHEMA = CONFIG["schema"]
 
 
+def log_concepts(session_id: int, concept_ids: List[int]):
```

Comment:
## Log concepts
Just curious - it says 'log', but it doesn't look like it's logging; it's adding to a session. Should this be renamed, like `add_concepts_to_session()`?